src/deploy/database/prepare.ts

```diff
@@ -1,28 +1,27 @@
-"use strict";
+import * as _ from "lodash";
+import * as clc from "cli-color";
+import * as path from "path";
 
-var _ = require("lodash");
-var clc = require("cli-color");
-var path = require("path");
+import { FirebaseError } from "../../error";
+import * as parseBoltRules from "../../parseBoltRules";
+import * as rtdb from "../../rtdb";
+import * as utils from "../../utils";
+import { Options } from "../../options";
 
-var { FirebaseError } = require("../../error");
-var parseBoltRules = require("../../parseBoltRules");
-var rtdb = require("../../rtdb");
-var utils = require("../../utils");
+import * as dbRulesConfig from "../../database/rulesConfig";
 
-const dbRulesConfig = require("../../database/rulesConfig");
-
-module.exports = function (context, options) {
-  var rulesConfig = dbRulesConfig.getRulesConfig(context.projectId, options);
-  var next = Promise.resolve();
+export function prepare(context: any, options: Options): Promise<any> {
```

Comment:
I think this can return Promise<void>, no?

Also, optionally, the code could be made quite a bit cleaner if this was an asyc function

Reply:
This does return a promise. It just doesn't use `async/await` - is that what you meant?